sys-boot/plymouth/plymouth-24.004.60.ebuild

```diff
@@ -97,14 +99,27 @@ src_install() {
 		dosym ../usr/sbin/plymouthd /sbin/plymouthd
 	fi
 
+	# the file /var/spool/plymouth/boot.log is linked to the boot log after
+	# booting when there are errors, there is no runtime check to create this
+	# directory (the file is directly linked using unistd `link`) meaning there
+	# may be missing logs or unexpected behaviour if it is not kept.
+	keepdir /var/spool/plymouth
+	# /var/lib/plymouth is created at runtime, and is used to store boot/shutdown
+	# durations, it doesn't need to be created at build.
+	rm -rf "${ED}"/var/lib || die
+	# /run/plymouth is also created at runtime
+	rm -rf "${ED}"/run || die
+
 	readme.gentoo_create_doc
 }
 
 pkg_postinst() {
 	readme.gentoo_print_elog
 	if ! has_version "sys-kernel/dracut" && ! has_version "sys-kernel/genkernel-next[plymouth]"; then
-		ewarn "Dracut is not installed, you will need an initramfs with plymouth"
-		ewarn "support, you can emerge 'sys-kernel/dracut' or read the plymouth"
-		ewarn "wiki at: https://wiki.gentoo.org/wiki/Plymouth#Building_Initramfs"
+		ewarn "Neither dracut or genkernel-next[plymouth] are installed"
+		ewarn "if you wish to have an initramfs with plymouth support, you can"
+		ewarn "emerge 'sys-kernel/dracut' or genkernel-next[plymouth] otherwise"
+		ewarn "you can look at the plymouth wiki for other methods:"
+		ewarn "https://wiki.gentoo.org/wiki/Plymouth#Building_Initramfs"
```

Comment:
I don't know much about genkernel, but my understanding is that genkernel-next has been merged into genkernel and/or discontinued, so I think this `has_version` is outdated and depending on the status of plymouth support in genkernel it should either be reworded or removed.

Reply:
done!

Reply:
set the `genkernel-next` stuff as outdated on the plymouth wiki

Reply:
Thanks